tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -105,11 +105,11 @@
   // CHECK-NEXT: %minf = arith.minf %lhsf32, %rhsf32 : f32
   // CHECK-NEXT: %minf_vector = arith.minf %lhsvec, %rhsvec : vector<4xf32>
 
-  %addf = arith.addf %lhsf32, %rhsf32 : f32
-  %addf_vector = arith.addf %lhsvec, %rhsvec : vector<4xf32>
+  %addf = arith.addf %lhsf32, %rhsf32 fastmath<fast> : f32
```

Comment:
feels like these lines should be in addition to the existing ones

Reply:
Sure, can do